src/o365/spo/commands/get.spec.ts

```diff
@@ -0,0 +1,179 @@
+import commands from '../commands';
+import Command, { CommandOption, CommandValidate, CommandError } from '../../../Command';
+import * as sinon from 'sinon';
+import appInsights from '../../../appInsights';
+const command: Command = require('./get');
+import * as assert from 'assert';
+import Utils from '../../../Utils';
+import auth from '../../../Auth';
+
+describe(commands.WEB_GET, () => {
```

Comment:
`WEB_GET` changed to just `GET` to reflect the name of actual command to be tested